.ci/integrationTestSelector.groovy

```diff
@@ -51,29 +52,80 @@ pipeline {
     /**
       launch integration tests.
     */
-    stage("Integration Tests"){
+    stage('Integration Tests'){
+      options { skipDefaultCheckout() }
       environment {
         TMPDIR = "${WORKSPACE}"
         ENABLE_ES_DUMP = "true"
         PATH = "${WORKSPACE}/${BASE_DIR}/.ci/scripts:${env.PATH}"
         APP = agentMapping.app(params.AGENT_INTEGRATION_TEST)
+        OPBEANS_APP = agentMapping.opbeansApp(params.AGENT_INTEGRATION_TEST)
```

Comment:
There is a match between agents and opbeans repos, but the `Node.js` opbeans app is called `node` instead.

Reply:
https://github.com/elastic/apm-pipeline-library/issues/398 could help?

Reply:
It's now solved with the maps, but wanted to clarify one of the exceptions for avoid using the agent name